{{ .ProjectSnake }}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+- repo: local
+  hooks:
+  - id: aspect-format
+    name: aspect-format
+    entry: aspect run format
+    language: system
+    pass_filenames: false
+    types: [text]
```

Comment:
hm, do you know how this differs from `files: .*` ?

See recommendation for installing: https://github.com/aspect-build/rules_lint/blob/main/docs/formatting.md#install-as-a-pre-commit-hook

Reply:
Based on https://pre-commit.com/#filtering-files-with-types, binary files would be ignored with this configuration, whereas with `files: .*` they'd be passed through.

Thanks for linking to those docs, I hadn't seen them. I notice another difference with them is that they specify the command `bazel run //:format`. Here I used `aspect run format` because that's what you ran in [your demo video](https://www.youtube.com/watch?v=ge9udVnj_wo). I guess they both work, but the `aspect` spelling is better for interactive use, while the `bazel` spelling is better for a pre-commit config, so pre-commit could be run in CI and the build user would only need `bazel` (not also `aspect`) in its PATH?

Reply:
Mostly true. However users might choose to put only 'aspect' on their $PATH rather than run an installer like brew. No answer that works in 100% of cases. I think this repo should just be consistent on 'aspect'